app/src/main/java/org/oppia/android/app/drawer/NavigationDrawerFragmentPresenter.kt

```diff
@@ -278,24 +278,24 @@ class NavigationDrawerFragmentPresenter @Inject constructor(
   }
 
   fun checkLastCheckedItemAndCloseDrawer(
-    lastCheckedItemId: Int,
+    lastCheckedMenuItemId: Int,
     isAdministratorControlsSelected: Boolean
   ) {
     if (isAdministratorControlsSelected) {
       getFooterViewModel().isAdministratorControlsSelected.set(true)
       uncheckAllMenuItemsWhenAdministratorControlsIsChecked()
-    } else if (lastCheckedItemId != -1) {
+    } else if (lastCheckedMenuItemId != -1) {
       getFooterViewModel().isAdministratorControlsSelected.set(false)
-      val checkedItemPosition =
-        when (lastCheckedItemId) {
+      val checkedMenuItemPosition =
+        when (lastCheckedMenuItemId) {
           NavigationDrawerItem.HOME.value -> 0
           NavigationDrawerItem.OPTIONS.value -> 1
           NavigationDrawerItem.HELP.value -> 2
           NavigationDrawerItem.DOWNLOADS.value -> 3
```

Comment:
So now that we're using a proto, could we also use an enum to represent the last checked menu item? That provides type safety vs. passing an integer around.

We may be able to share the draw item enum & the menu item enum as well (e.g. just have 1 proto enum to define both so that we can include it in argument protos).

Reply:
Done.